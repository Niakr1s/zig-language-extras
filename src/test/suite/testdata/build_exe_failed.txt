zig build-exe spell_checker Debug native: error: the following command failed with 1 compilation errors:
/usr/local/zig/zig-linux-aarch64-0.11.0-dev.3886+0c1bfe271/zig build-exe /home/ianic/zig/zds/examples/spell_checker.zig --cache-dir /home/ianic/zig/zds/zig-cache --global-cache-dir /home/ianic/.cache/zig --name spell_checker --listen=- 
Build Summary: 2/5 steps succeeded; 1 failed (disable with --summary none)
install transitive failure
+- install spell_checker transitive failure
   +- zig build-exe spell_checker Debug native 1 errors
/usr/local/zig/zig-linux-aarch64-0.11.0-dev.3886+0c1bfe271/lib/std/start.zig:593:45: error: 'main' is not marked 'pub'
examples/spell_checker.zig:4:1: note: declared here
/usr/local/zig/zig-linux-aarch64-0.11.0-dev.3886+0c1bfe271/lib/std/start.zig:542:12: note: called from here
/usr/local/zig/zig-linux-aarch64-0.11.0-dev.3886+0c1bfe271/lib/std/start.zig:492:36: note: called from here
/usr/local/zig/zig-linux-aarch64-0.11.0-dev.3886+0c1bfe271/lib/std/start.zig:455:17: note: called from here