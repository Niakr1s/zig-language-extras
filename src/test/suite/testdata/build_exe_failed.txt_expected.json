[
  {
    "file": "/usr/local/zig/zig-linux-aarch64-0.11.0-dev.3886+0c1bfe271/lib/std/start.zig",
    "line": 592,
    "column": 44,
    "message": "'main' is not marked 'pub'",
    "severity": 0
  },
  {
    "file": "/project/root/examples/spell_checker.zig",
    "line": 3,
    "column": 0,
    "message": "'main' is not marked 'pub'\ndeclared here",
    "severity": 2
  },
  {
    "file": "/usr/local/zig/zig-linux-aarch64-0.11.0-dev.3886+0c1bfe271/lib/std/start.zig",
    "line": 541,
    "column": 11,
    "message": "'main' is not marked 'pub'\ncalled from here",
    "severity": 2
  },
  {
    "file": "/usr/local/zig/zig-linux-aarch64-0.11.0-dev.3886+0c1bfe271/lib/std/start.zig",
    "line": 491,
    "column": 35,
    "message": "'main' is not marked 'pub'\ncalled from here",
    "severity": 2
  },
  {
    "file": "/usr/local/zig/zig-linux-aarch64-0.11.0-dev.3886+0c1bfe271/lib/std/start.zig",
    "line": 454,
    "column": 16,
    "message": "'main' is not marked 'pub'\ncalled from here",
    "severity": 2
  }
]